test/suite/hook_tiered.py

```diff
@@ -393,6 +393,16 @@ def setup_hooks(self):
 
 # Override some platform APIs for this hook.
 class TieredPlatformAPI(wthooks.WiredTigerHookPlatformAPI):
+    def __init__(self, arg=0):
+        self.tier_share_percent = 0
+        params = []
+        if arg is not None:
+            params = [config.split('=') for config in arg.split(',')]
+
+        for param_key, param_value in params :
+            if param_key == 'tier_populate_share':
+                self.tier_share_percent = param_value
```

Comment:
This is awesome Pythonic code!

Except for a couple things.  First a nit, I would change to simply `if arg:`.  We're taught never to do that in C, but it's more idiomatic for Python.  Second, I would convert param_value to an int right here -- `....share_percent = int(param_value)`.  Then it's already in a usable form for the consumer of the API.

Reply:
Thanks for the suggestion @ddanderson . Fixed.